src/codewhisperer/commands/startTransformByQ.ts

```diff
@@ -193,6 +203,12 @@ export async function startTransformByQ() {
         } catch (error) {
             errorMessage = 'Failed to start job'
             getLogger().error(errorMessage, error)
+            telemetry.codeTransform_logGeneralError.emit({
+                codeTransformSessionId: codeTransformTelemetryState.getSessionId(),
+                codeTransformApiErrorMessage: errorMessage,
```

Comment:
Was is confirmed that error message doesn't have in PII information?

Reply:
I think these logs are redundant so will remove them. They should not have contained PII.